.vscode/settings.json

```diff
@@ -94,17 +96,6 @@
     "@o3r/core:*"
   ],
   "otter.extract.styling.prefix": "o3r",
-  "search.exclude": {
```

Comment:
Why do you remove this field ?

Reply:
If not specified, it inherits of `files.exclude` (which include git ignored files).
The added value of specifying it is for `.yarn/release/yarn.cjs` and `.yarn/sdks` but we will lose the benefit of relying on `files.exclude`

Reply:
edit: After testing, with the option `search.useIgnoreFiles` explicitly set, the `search.exclude` property extends the ignored files.
So I put it back with a minimized value